packages/components/src/index.jsx

```diff
@@ -0,0 +1,3 @@
+export { default as Dots } from './dots';
```

Comment:
would be better to autogenerate this file. ideas?

Reply:
I used a plugin to autogenerate those imports. As long as every folder exports one default-export - it's easy but Components like MicroHelmet export default and Provider

Therefore I switched over to the manual approach for now.  Any idea on how to handle these cases?

Reply:
`export * as camelCase, { default as SnakeCase } from 'whatever';`

would something like this work? (play around with syntax, this is probably incorrect)